src/component/page/tiltaksoversikt/bruker-oversikt/TabellHeader.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+export enum SortDirection {
+	ASCENDING = 'ascending',
+	DESCENDING = 'descending',
+	NONE = 'none'
+}
+
+export enum TableHeaderName {
+	NAVN = 'NAVN',
+	FODSELSDATO = 'FODSELSDATO',
+	TILTAKSTYPE = 'TILTAKSTYPE',
+	TILTAK = 'TILTAK',
+	STATUS = 'STATUS',
+	START = 'START',
+	SLUTT = 'SLUTT'
+}
+
+export interface BrukerSortering {
```

Comment:
Denne blir brukt litt på kryss og tvers nå så burde flyttes ut. Vi kan for eksempel ha en mappe med domeneobjekter